ga4gh/datamodel/variants.py

```diff
@@ -490,9 +506,42 @@ def _addFile(self, filename):
                 if chrom in self._chromFileMap:
                     raise Exception("cannot have overlapping VCF/BCF files.")
                 self._updateMetadata(varFile)
+                self._updateCallSetIds(varFile)
                 self._chromFileMap[chrom] = varFile
 
-    def convertVariant(self, record):
+    def convertVariant(self, record, callSetIds):
+        """
+        Converts the specified pysam variant record into a GA4GH Variant
+        object. Only calls for the specified list of callSetIds will
+        be included.
+        """
+        def _convertGaCall(recordId, recordInfo, call):
```

Comment:
I think convertCall is probably important enough to get a top-level method. These local functions are useful at times, but are a little over-used here. As a rule of thumb, if you're not actually using the context of the surrounding function then it should probably be a normal method.